config/application.rb

```diff
@@ -54,5 +54,27 @@ class Application < Rails::Application
 
       Object.include(AccountSwitch)
     end
+
+    ##
+    # Psych Allow YAML Classes
+    #
+    # The following configuration addresses errors of the following form:
+    #
+    # ```
+    # Psych::DisallowedClass: Tried to load unspecified class: ActiveSupport::HashWithIndifferentAccess
+    # ```
+    #
+    # Psych::DisallowedClass: Tried to load unspecified class: <Your Class Name Here>
+    config.after_initialize do
+      config.active_record.yaml_column_permitted_classes = [
```

Comment:
Why `after_initialize`? I don't see that in ESSI's solution

Reply:
it makes sure all the models are loaded in prod mode before running